staking-pool/tests/utils.rs

```diff
@@ -220,6 +265,47 @@ pub fn init_pool(initial_transfer: Balance) -> (RuntimeStandalone, ExternalUser)
             numerator: 10,
             denominator: 100,
         },
-    );
+    )
+    .unwrap();
     return (runtime, root);
 }
+
+pub fn reward_pool(runtime: &mut RuntimeStandalone, amount: Balance) {
+    let mut pool_account = runtime.view_account(&POOL_ACCOUNT_ID.into()).unwrap();
+    pool_account.locked += amount;
+    runtime.force_account_update(POOL_ACCOUNT_ID.into(), &pool_account);
+}
+
+pub fn wait_epoch(runtime: &mut RuntimeStandalone) {
+    let epoch_height = runtime.current_block().epoch_height;
```

Comment:
Isn't this always one block?

Reply:
Epoch height is different from block heights. One epoch might be half a day of blocks

Reply:
Yeah I just thought in the RunnerStandalone it was each block.